index.js

```diff
@@ -24,7 +24,7 @@ module.exports = {
 	],
 	settings: {
 		'import/resolver': {
-			node: {
+			n: {
```

Comment:
n? is that for disabling or is that some magic syntax ?

Reply:
the plugin is called "n" instead of "node" -> eslint-plugin-n